packages/flutter_availability/lib/src/ui/screens/availability_overview.dart

```diff
@@ -135,53 +137,25 @@ class _AvailabilityOverviewState extends State<AvailabilityOverview> {
       Text(translations.editAvailabilityButton),
     );
 
-    // TODO(Joey): This structure is defined multiple times
-    var body = CustomScrollView(
-      slivers: [
-        SliverPadding(
-          padding: EdgeInsets.symmetric(horizontal: spacing.sidePadding),
-          sliver: SliverList.list(
-            children: [
-              const SizedBox(height: 40),
-              title,
-              const SizedBox(height: 24),
-              calendar,
-              const SizedBox(height: 32),
-              templateLegend,
-              const SizedBox(height: 32),
-            ],
-          ),
-        ),
-        SliverFillRemaining(
-          fillOverscroll: false,
-          hasScrollBody: false,
-          child: Padding(
-            padding: EdgeInsets.symmetric(
-              horizontal: spacing.sidePadding,
-            ).copyWith(
-              bottom: spacing.bottomButtonPadding,
-            ),
-            child: Align(
-              alignment: Alignment.bottomCenter,
-              child: Column(
-                children: [
-                  startEditButton,
-                  if (availabilitiesAreSelected) ...[
-                    const SizedBox(height: 8),
-                    clearSelectedButton,
-                  ],
-                ],
-              ),
-            ),
-          ),
-        ),
-      ],
-    );
-
     return options.baseScreenBuilder(
       context,
       widget.onExit,
-      body,
+      BasePage(
```

Comment:
extract this to a variable

Reply:
What would be the point of that? Most of this has already been extracted, I see no use at all in moving this to a variable.